magic/multiverse.py

```diff
@@ -259,274 +258,3 @@ def card_name(c):
-        'colors': ['White']
-    }
-    cards['Initiate of Blood'] = {
-        'text': '{T}: Initiate of Blood deals 1 damage to target creature that was dealt damage this turn. When that creature dies this turn, flip Initiate of Blood.',
-        'manaCost': '{3}{R}',
-        'type': 'Creature — Ogre Shaman',
-        'power': '2',
-        'layout': 'flip',
-        'names': ['Initiate of Blood', 'Goka the Unjust'],
-        'types': ['Creature'],
-        'colorIdentity': ['R'],
-        'toughness': '2',
-        'cmc': 4,
-        'imageName': 'intitiate of blood',
-        'legalities': [
-            {'format': 'Commander', 'legality': 'Legal'},
-            {'format': 'Kamigawa Block', 'legality': 'Legal'},
-            {'format': 'Legacy', 'legality': 'Legal'},
-            {'format': 'Modern', 'legality': 'Legal'},
-            {'format': 'Vintage', 'legality': 'Legal'}],
-        'name': 'Initiate of Blood',
-        'subtypes': ['Ogre', 'Shaman'],
-        'printings': ['COK'],
-        'colors': ['Red']
-    }
-    cards['Akki Lavarunner'] = {
-        'text': 'Haste\nWhenever Akki Lavarunner deals damage to an opponent, flip it.',
-        'manaCost': '{3}{R}',
-        'type': 'Creature — Goblin Warrior',
-        'power': '1',
-        'layout': 'flip',
-        'names': ['Akki Lavarunner', 'Tok-Tok, Volcano Born'],
-        'types': ['Creature'],
-        'colorIdentity': ['R'],
-        'toughness': '1',
-        'cmc': 4,
-        'imageName': 'akki lavarunner',
-        'legalities': [
-            {'format': 'Commander', 'legality': 'Legal'},
-            {'format': 'Kamigawa Block', 'legality': 'Legal'},
-            {'format': 'Legacy', 'legality': 'Legal'},
-            {'format': 'Modern', 'legality': 'Legal'},
-            {'format': 'Vintage', 'legality': 'Legal'}],
-        'name': 'Akki Lavarunner',
-        'subtypes': ['Ogre', 'Shaman'],
-        'printings': ['COK'],
-        'colors': ['Red']
-    }
-    cards["Budoka Gardener"] = {
-        "text": "{T}: You may put a land card from your hand onto the battlefield. If you control ten or more lands, flip Budoka Gardener.",
-        "manacost": "{1}{G}",
-        "type": "Creature — Human Monk",
-        "power": "2",
-        "layout": "flip",
-        "names": ["Budoka Gardener", "Dokai, Weaver of Life"],
-        "types": ["Creature"],
-        'colorIdentity': ['G'],
-        "toughness": "1",
-        "cmc": 2,
-        'imageName': 'budoka gardener',
-        "legalities": [
-            {"format": "Modern", "legality": "Legal"},
-            {"format": "Kamigawa Block", "legality": "Legal"},
-            {"format": "Legacy", "legality": "Legal"},
-            {"format": "Vintage", "legality": "Legal"},
-            {"format": "Commander", "legality": "Legal"}],
-        "name": "Budoka Gardener",
-        "subTypes": ["Human", "Monk"],
-        "printings": ["CHK"],
-        "colors": ["Green"],
-        "rarity": "Rare"
-    }
-    cards["Gleemox"] = {
```

Comment:
One of these is not like the others.
We need to hardcode Gleemox because it isn't (and won't be) added to mtgjson.